asset/src/kafka_sender/schema.ts

```diff
@@ -23,14 +29,25 @@ export default class Schema extends ConvictSchema<KafkaSenderConfig> {
                 }
             }
         }
+
+        const apiName = opConfig?.api_name;
+
+        if (isNotNil(apiName)) {
+            const kafkaSenderAPI = job.apis.find((api) => api._name === apiName);
+            if (isNil(kafkaSenderAPI)) throw new Error(`kafka_sender parameter for api_name: "${kafkaSenderAPI}" was not found listed in the apis of this execution ${JSON.stringify(job, null, 4)}`);
```

Comment:
We should log the entire job schema here, it make the error unreadable

```suggestion
            if (isNil(kafkaSenderAPI)) throw new Error(`kafka_sender parameter for api_name: "${kafkaSenderAPI}" was not found listed in the apis`);
```